hc/front/tests/test_add_channel.py

```diff
@@ -38,4 +38,21 @@ def test_instructions_work(self):
             self.assertContains(r, "Integration Settings", status_code=200)
 
     ### Test that the team access works
+    def test_team_access(self):
+        """ login as a different user"""
+        self.client.login(username='bob@example.org', password="password")
+
+        """ a url to make a request to alice's stuff using Bobs's credential who is a team member"""
+        url = "/accounts/switch_team/alice"
+        response = self.client.get(url)
+        self.assertNotEqual(response.status_code, 403)
+
     ### Test that bad kinds don't work
+    def test_bad_kinds(self):
+        self.client.login(username='bob@example.org', password="password")
+        """ provide non existent kinds"""
```

Comment:
the triple quotes is put at the beginning of a class .. Single line comments are with hash